tsconfig.base.json

```diff
@@ -7,6 +7,7 @@
     "moduleResolution": "Node",
     "noEmit": true,
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
We're currently seeing errors raised in:

```console
@jest/environment
@types/glob-stream
@types/glob
@types/jest
path-scurry
```

Typically this is where packages have shipped their own types, leaving the community types out of sync in various unmaintained packages